src/styles/star.css

```diff
@@ -0,0 +1,16 @@
+.card__star {
+  display: flex;
+  margin: 0 16px;
+  font-size: 10px;
+}
+
+.stars_star {
+  background: url(../images/star.svg) no-repeat center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars2 .stars_star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
i think, you didn`t this point of the task.

Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. **If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.**

you didn't add styles for other modifiers